src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -110,55 +110,31 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
   const isDeposit = isLpConfirmedDepositQuote(confirmedQuote)
   const isSymWithdraw = isLpConfirmedWithdrawalQuote(confirmedQuote) && opportunityType === 'sym'
 
-  const accountId = useMemo(
-    () => (isRuneTx ? runeAccountId : poolAssetAccountId),
-    [isRuneTx, poolAssetAccountId, runeAccountId],
-  )
-
-  const accountMetadata = useMemo(
-    () => (isRuneTx ? runeAccountMetadata : poolAssetAccountMetadata),
-    [isRuneTx, poolAssetAccountMetadata, runeAccountMetadata],
-  )
-
-  const pairAssetAssetId = useMemo(
-    () => (isRuneTx ? poolAssetId : thorchainAssetId),
-    [isRuneTx, poolAssetId],
-  )
-
-  const pairAssetAccountMetadata = useMemo(
-    () => (isRuneTx ? poolAssetAccountMetadata : runeAccountMetadata),
-    [isRuneTx, poolAssetAccountMetadata, runeAccountMetadata],
-  )
-
   const { data: fromAddress } = useQuery({
     ...reactQueries.common.thorchainFromAddress({
-      accountId,
+      accountId: isRuneTx ? runeAccountId : poolAssetAccountId,
       assetId: isRuneTx ? thorchainAssetId : poolAssetId,
       opportunityId: confirmedQuote.opportunityId,
       wallet: wallet!,
-      accountMetadata: accountMetadata!,
+      accountMetadata: isRuneTx ? runeAccountMetadata! : poolAssetAccountMetadata!,
       getPosition: getThorchainLpPosition,
     }),
-    enabled: Boolean(poolAsset?.assetId && accountMetadata && wallet && poolAssetAccountMetadata),
+    enabled: Boolean(poolAssetAccountMetadata && runeAccountMetadata && wallet),
```

Comment:
@kaladinlight shouldn't it be `poolAssetAccountMetadata || runeAccountMetadata` i.e for Ledger with only one of the two added?

Reply:
i.e seems like the above could be kept, but we only want to remove he `poolAssetaccountMetadata` check from the `enabled`

Reply:
Good thinking, will pull those back out into memo vars with better naming this time so they don't get mixed up

Reply:
a7539104b7729a2f07664a9ee0e15d11f22b995e